### PR TITLE
Clarify that -Zfile reads from last data column

### PR DESCRIPTION
**Pull request by PaulWessel**
Also improve the documentation so it is clear that either _value_ or _file_ is acceptable.

### CODE REVIEW

File: doc/rst/source/plot.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ Synopsis
 [ |-W|\ [*pen*][*attr*] ]
 [ |SYN_OPT-X| ]
 [ |SYN_OPT-Y| ]
-[ |-Z|\ *value* [**+f**\|\ **l**]]
+[ |-Z|\ *value*\ \|\ *file*\ [**+f**\|\ **l**]]
```
Comment (seisman) [inline doc/rst/source/plot.rst]:
```suggestion
[ |-Z|\ *value*\|\ *file*\ [**+f**\|\ **l**]]
```

File: doc/rst/source/plot3d.rst [hunk 2]
```diff
@@ -29,7 +29,7 @@ Synopsis
 [ |-W|\ [*pen*][*attr*] ]
 [ |SYN_OPT-X| ]
 [ |SYN_OPT-Y| ]
-[ |-Z|\ *value* [**+f**\|\ **l**]]
+[ |-Z|\ *value*\ \|\ *file*\ [**+f**\|\ **l**]]
```
Comment (seisman) [inline doc/rst/source/plot3d.rst]:
```suggestion
[ |-Z|\ *value*\|\ *file*\ [**+f**\|\ **l**]]
```

File: doc/rst/source/plot3d_common.rst_ [hunk 3]
```diff
@@ -141,10 +141,10 @@ Optional Arguments
 
 .. _-Z:
 
-**-Z**\ *value* [**+f**\|\ **l**]
+**-Z**\ *value*\ \|\ *file*\ [**+f**\|\ **l**]
```
Comment (seisman) [inline doc/rst/source/plot3d_common.rst_]:
```suggestion
**-Z**\ *value*\|\ *file*\ [**+f**\|\ **l**]
```

File: doc/rst/source/plot_common.rst_ [hunk 4]
```diff
@@ -192,10 +192,10 @@ Optional Arguments
 
 .. _-Z:
 
-**-Z**\ *value* [**+f**\|\ **l**]
+**-Z**\ *value*\ \|\ *file*\ [**+f**\|\ **l**]
```
Comment (seisman) [inline doc/rst/source/plot_common.rst_]:
```suggestion
**-Z**\ *value*\|\ *file*\ [**+f**\|\ **l**]
```

**Comment by PaulWessel** [comment]
Thanks for cleaning up my mess, @seisman!

**Pull request merged**